README.md

```diff
@@ -87,7 +87,7 @@ Here is two example.
 ```gradle
 unusedResourcesRemover { 
   extraRemovers = [
-    createXmlValueRemover("font", "string", "string"), // fonts.xml
+    createXmlValueRemover("fonts", "string", "string"), // fonts.xml
```

Comment:
Actually this was expected. I thought some users are using `font-xxx.xml`.
But `fonts` would be easier to understand as you fixed!